completions/pyenv.fish

```diff
@@ -18,5 +18,6 @@ end
 
 complete -f -c pyenv -n '__fish_pyenv_needs_command' -a '(pyenv commands)'
 for cmd in (pyenv commands)
-  complete -f -c pyenv -n "__fish_pyenv_using_command $cmd" -a "(pyenv completions $cmd)"
+  complete -f -c pyenv -n "__fish_pyenv_using_command $cmd" -a \
+    "(pyenv completions (commandline -opc)[2..-1])"
```

Comment:
That is already handled in the `__fish_pyenv_needs_command`/`__fish_pyenv_using_command `, but if it was just one completion then yes.